src/machine/board_grandcentral-m4.go

```diff
@@ -0,0 +1,244 @@
+	D96 = PB30 // SWO                                        EXTI14
+)
+
+// Analog pins
+const (
+	A0  = D67 // (PA02) ADC0 ch. 0,
+	A1  = D68 // (PA05) ADC0 ch. 5,
+	A2  = D69 // (PB03) ADC0 ch. 15
+	A3  = D70 // (PC00) ADC1 ch. 10
+	A4  = D71 // (PC01) ADC1 ch. 11
+	A5  = D72 // (PC02) ADC1 ch. 4
+	A6  = D73 // (PC03) ADC1 ch. 5
+	A7  = D74 // (PB04) ADC1 ch. 6
+	A8  = D54 // (PB05) ADC1 ch. 7
+	A9  = D55 // (PB06) ADC1 ch. 8
+	A10 = D56 // (PB07) ADC1 ch. 9
+	A11 = D57 // (PB08) ADC1 ch. 0
+	A12 = D58 // (PB09) ADC1 ch. 1
+	A13 = D59 // (PA04) ADC0 ch. 4
+	A14 = D60 // (PA06) ADC0 ch. 6
+	A15 = D61 // (PA07) ADC0 ch. 7
+
+	AREF = D84 // (PA03)
+)
+
+// LED pins
+const (
+	LED_PIN         = D13 // (PB01), also on D87
+	UART_RX_LED_PIN = D75 // (PC31)
+	UART_TX_LED_PIN = D76 // (PC30)
+	NEOPIXEL_PIN    = D88 // (PC24)
+
+	// aliases used by examples and drivers
+	LED      = LED_PIN
+	LED_RX   = UART_RX_LED_PIN
+	LED_TX   = UART_TX_LED_PIN
+	NEOPIXEL = NEOPIXEL_PIN
+)
+
+// UART pins
+const (
+	UART1_RX_PIN = D0 // (PB25)
+	UART1_TX_PIN = D1 // (PB24)
+
+	UART2_RX_PIN = D19 // (PB13)
+	UART2_TX_PIN = D18 // (PB12)
+
+	UART3_RX_PIN = D17 // (PC23)
+	UART3_TX_PIN = D16 // (PC22)
+
+	UART4_RX_PIN = D15 // (PB17)
+	UART4_TX_PIN = D14 // (PB16)
+
+	UART_RX_PIN = UART1_RX_PIN // default pins
+	UART_TX_PIN = UART1_TX_PIN //
+)
+
+// SPI pins
+const (
+	SPI0_SCK_PIN = D66 // (PD09), also on D52
+	SPI0_SDO_PIN = D65 // (PD08), also on D51
+	SPI0_SDI_PIN = D64 // (PD11), also on D50
+	SPI0_CS_PIN  = D53 // (PD10)
+
+	SPI1_SCK_PIN = D81 // (PB27)
+	SPI1_SDO_PIN = D82 // (PB26)
+	SPI1_SDI_PIN = D80 // (PB29)
+
+	SPI_SCK_PIN = SPI0_SCK_PIN // default pins
+	SPI_SDO_PIN = SPI0_SDO_PIN //
+	SPI_SDI_PIN = SPI0_SDI_PIN //
+	SPI_CS_PIN  = SPI0_CS_PIN  //
+)
+
+// I2C pins
+const (
+	I2C0_SDA_PIN = D62 // (PB20), also on D20
+	I2C0_SCL_PIN = D63 // (PB21), also on D21
+
+	I2C1_SDA_PIN = D25 // (PC16)
+	I2C1_SCL_PIN = D24 // (PC17)
+
+	I2C_SDA_PIN = I2C0_SDA_PIN // default pins
+	I2C_SCL_PIN = I2C0_SCL_PIN //
+
+	SDA_PIN = I2C_SDA_PIN // unconventional pin names
+	SCL_PIN = I2C_SCL_PIN //  (required by machine_atsamd51.go)
+)
+
+// I2S pins
+const (
+	I2S0_SCK_PIN = D14 // (PB16)
+	I2S0_MCK_PIN = D15 // (PB17)
+	I2S0_FS_PIN  = D33 // (PA20)
+	I2S0_SDO_PIN = D32 // (PA21)
+	I2S0_SDI_PIN = D31 // (PA22)
+
+	I2S_SCK_PIN = I2S0_SCK_PIN // default pins
+	I2S_WS_PIN  = I2S0_FS_PIN  //
+	I2S_SD_PIN  = I2S0_SDO_PIN //
+)
+
+// SD card pins
+const (
+	SD0_SCK_PIN = D81 // (PB27)
+	SD0_SDO_PIN = D82 // (PB26)
+	SD0_SDI_PIN = D80 // (PB29)
+	SD0_CS_PIN  = D83 // (PB28)
+	SD0_DET_PIN = D95 // (PB31)
+
+	SDCARD_SCK_PIN = SD0_SCK_PIN // default pins
+	SDCARD_SDO_PIN = SD0_SDO_PIN //
+	SDCARD_SDI_PIN = SD0_SDI_PIN //
+	SDCARD_CS_PIN  = SD0_CS_PIN  //
+	SDCARD_DET_PIN = SD0_DET_PIN //
+)
+
+// Other peripheral constants
+const (
+	RESET_MAGIC_VALUE      = 0xF01669EF // Used to reset into bootloader
+	EXTERNAL_FLASH_DEVICES = "GD25Q16C" // QSPI
```

Comment:
`GD25Q64` seems to be correct.
https://cdn-learn.adafruit.com/assets/assets/000/069/175/original/adafruit_products_Grand_Central_Sch1.png?1547234817